<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>University Faclilities Management System (UFMS)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="btn_register_reg">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>406</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="Line" name="oframe3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="copyright">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>570</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Â© SLAUATRON 2019.</string>
   </property>
  </widget>
  <widget class="Line" name="oframe2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="oframe4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="oframe1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username_reg">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>224</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USERNAME:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password_reg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>256</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>258</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PASSWORD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create an account to access to UFMS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="seq_comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>292</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>What is your favorite color?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Where is your primary school?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What is your first pet's name?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Who is your favorite author?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What is your lucky number?</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_SEQues">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>294</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SECURITY QUESTION:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>186</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>STUDENT ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_studid">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>184</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rback2login">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BACK TO LOGIN PAGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Answer">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>330</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ANSWER:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_seqanswer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>328</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username cannot be the same as password!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="db_lbl">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>570</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected to UFMS!</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Student ID or username cannot be empty!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username is already been taken!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ID is already been taken!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ID and Username is already been taken!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lbl_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>User already existed!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_studid</tabstop>
  <tabstop>lineEdit_username_reg</tabstop>
  <tabstop>lineEdit_password_reg</tabstop>
  <tabstop>seq_comboBox</tabstop>
  <tabstop>lineEdit_seqanswer</tabstop>
  <tabstop>btn_register_reg</tabstop>
  <tabstop>btn_rback2login</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_studid</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_username_reg</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_username_reg</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_password_reg</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_password_reg</sender>
   <signal>returnPressed()</signal>
   <receiver>seq_comboBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_seqanswer</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_register_reg</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
